pkg/apis/greenhouse/v1alpha1/cluster_types.go

```diff
@@ -29,6 +29,12 @@ const (
 
 	// KubeConfigValid reflects the validity of the kubeconfig of a cluster.
 	KubeConfigValid ConditionType = "KubeConfigValid"
+
+	// ClusterDeletionScheduledReason reflects the reason if a cluster is scheduled for deletion
+	ClusterDeletionScheduledReason ConditionReason = "ClusterDeletionScheduled"
```

Comment:
We might not need this, if we do not reflect scheduled for deletion in `Ready` condition?

Reply:
yes, I have removed it